test/models/patient_test.rb

```diff
@@ -3097,6 +3114,77 @@ def verify_patient_status(patient, status)
     end
     ADMIN_OPTIONS['reporting_period_minutes'] = original_reporting_period
   end
+
+  test 'no_recent_activity scope in exposure' do
+    patient = create(:patient)
+    assert_nil Patient.no_recent_activity.find_by(id: patient.id)
+
+    patient.update(updated_at: 1.day.ago)
+    assert_nil Patient.no_recent_activity.find_by(id: patient.id)
+
+    patient.update(updated_at: 5.days.ago)
+    assert_nil Patient.no_recent_activity.find_by(id: patient.id)
+
+    patient.update(updated_at: 10.days.ago)
+    assert_nil Patient.no_recent_activity.find_by(id: patient.id)
+
+    patient.update(updated_at: 20.days.ago)
+    assert_nil Patient.no_recent_activity.find_by(id: patient.id)
+
+    patient.update(updated_at: 29.days.ago)
+    assert_nil Patient.no_recent_activity.find_by(id: patient.id)
+
+    # 30 day border is sensitive to DST changes
```

Comment:
Any thoughts on specific tests at DST edge?

Reply:
added dst edge test

Reply:
i was thinking a timecop test where we change the date to DST ? Am I missing something?

Reply:
We have the inherited test classes at the bottom of patient_test that run all of the patient tests on two DST edges

Reply:
I finally understand the comment. Took me a little bit.